app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1185,10 +1187,10 @@ function ThresholdMap() {
       name: "NT log(1/e)",
       value: "NT_neglogevalue"
     },
-    {
-      name: "NT %conc",
-      value: "NT_percentconcordant"
-    },
+    //{
```

Comment:
Instead of comment this out, 
check  this.showConcordance to decide to hide it or not?